### PR TITLE
Generate and add `ActiveCampaign` connector

### PR DESCRIPTION
**Pull request by sachinira**
# Description
ActiveCampaign is an integrated email marketing, automation, sales software, and CRM platform. It lets users perform powerful automation, email marketing, and customer relationship management. This is a  connector featured in most popular IPaaS vendors. For the generation of the connector OpenAPI file needs to be handwritten because it is not available publicly.

Fixes: https://github.com/ballerina-platform/ballerina-extended-library/issues/356

One line release note: 
-  Add `ActiveCampaign` connector

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
**Not Tested Yet**

**Test Configuration**:
* Ballerina Version:
* Operating System:
* Java SDK: 

# Checklist:

### Changes to OpenAPI definition

- [x] Modified `info` section of the OpenAPI definition file - [Example](https://github.com/ballerina-platform/ballerina-extended-library/discussions/74)

    - `description` - what the connector is about 
    - `x-ballerina-init-description` OpenAPI extension - connector initialization details 
    - `x-ballerina-display` OpenAPI extension - connector name and path to the connector icon 

- [x] Added proper description each API Key, if API Key authentication is defined in OpenAPI definition file 

### Changes to connector module

- [x] Added license header at the top of each .bal file. 
- [x] Added Package.md as per the guide [here](https://github.com/ballerina-platform/ballerina-extended-library/discussions/77)
- [x] Added Module.md for the module of the  connector as per the guide [here](https://github.com/ballerina-platform/ballerina-extended-library/discussions/78)
- [x] Added connector icon to the root of the connector. Icon needs to be a `png` of 200x200 px size with name `icon.png`
- [x] Added Ballerina.toml file with following information. For keywords refer to [guide](https://github.com/ballerina-platform/ballerina-extended-library/discussions/72)

### Verifying connector module

- [x] Compiled the connector successfully with the targeted ballerina version. 
- [ ] Conducted smoke tests on the connector (Recommended when OpenAPI definition is obtain from an unofficial source)

### Security checks
 - [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by LakshanSS — APPROVED** [review]

**Review by aneeshafedo — APPROVED** [review]

File: openapi/activecampaign/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["Marketing/Marketing Automation", "Cost/Freemium"]
+org = "ballerinax"
+name = "activecampaign"
+icon = "icon.png"
+distribution = "2201.0.1"
```
Comment (SkNuwanTissera) [inline openapi/activecampaign/Ballerina.toml:7]:
```suggestion
distribution = "2201.0.2"
```

File: openapi/activecampaign/Module.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+```
+
+### Step 2: Create a new connector instance
+You can now make the connection configuration using the <API_TOKEN>.
+```ballerina
+activecampaign:ApiKeysConfig apiKeyConfig = {
+    apiToken:"<API_TOKEN>"
+};
+
+activecampaign:Client baseClient = check new Client(clientConfig);
```
Comment (SkNuwanTissera) [inline openapi/activecampaign/Module.md:30]:
```suggestion
activecampaign:Client baseClient = check new (clientConfig);
```
Shouldn't it be like this?

File: openapi/activecampaign/Module.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+## Overview
+This is a generated connector for [ActiveCampaign API version 3](https://developers.activecampaign.com/reference/overview) OpenAPI specification.
```
Comment (SkNuwanTissera) [inline openapi/activecampaign/Module.md:2]:
```suggestion
This is a generated connector for [ActiveCampaign API v3](https://developers.activecampaign.com/reference/overview) OpenAPI specification.
```

File: openapi/activecampaign/client.bal [hunk 4]
```diff
@@ -0,0 +1,460 @@
...
+import ballerina/http;
+
+# Provides API key configurations needed when communicating with a remote HTTP endpoint.
+public type ApiKeysConfig record {|
+    # Provide your API Key as Api-Token. Eg: {apiToken : <Your API Key>} Your API key can be found in your account on the Settings page under the Developer tab.
+    string apiToken;
+|};
+
+# This is a generated connector for [ActiveCampaign API version 3](https://developers.activecampaign.com/reference/overview) OpenAPI specification.  ActiveCampaignis an integrated email marketing, automation, sales software, and CRM platform. It lets users perform powerful automation, email marketing, and customer relationship management. 
+@display {label: "ActiveCampaign", iconPath: "icon.png"}
```
Comment (SkNuwanTissera) [inline openapi/activecampaign/client.bal:25]:
```suggestion
@display {label: "Active Campaign", iconPath: "icon.png"}
```

**Pull request merged**